src/Geno/C++/GUI/Widgets/TextEdit.cpp

```diff
@@ -448,7 +452,7 @@ bool TextEdit::RenderEditor( File& rFile )
 				if( !HasSelection( rFile, j ) && ( rFile.CursorMultiMode == MultiCursorMode::Normal || rFile.Cursors.size() == 1 ) )
 				{
 					ImVec2 Start( ScreenCursor.x + Props.LineNumMaxWidth - 2, Pos.y );
-					ImVec2 End( ScreenCursor.x + Size.x, Pos.y + Props.CharAdvanceY );
+					ImVec2 End( ScreenCursor.x + Size.x - ScrollBarSize, Pos.y + Props.CharAdvanceY );
```

Comment:
Can't we fetch the ScrollbarSize from the style directly, instead of storing a static float?

Reply:
Yes but I didn't know how to so I just did to and wait for you to tell me how

Reply:
lmao well it's just `ImGui::GetStyle().ScrollbarSize`. In fact you do it earlier in the code